intro-OpenStack/title.tex

```diff
@@ -31,9 +31,8 @@
 \pagestyle{plain}
 \strong{Audience:}
 
-This document is a hands-on guide for using the \gls{OpenStack} Cloud Computing platform in the
-context of the \strong{\university} \hpc Tier1 infrastructure. The resources are mostly deployed as infrastructure-as-a-service (IaaS).
-These resources can be managed either via a web-based UI (\gls{OpenStack Dashboard}), or using command line and scripts via the OpenStack API.
```

Comment:
I don't think regular users get access to the command line interface or the API (at least not at this moment?)